pkg/model/config/config.go

```diff
@@ -143,34 +158,62 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
+// Resource contains information about scaffolded resources
+type Resource struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
 
-	// CRDVersion holds the CustomResourceDefinition API version used for the GVK.
+	// CRDVersion holds the CustomResourceDefinition API version used for the Resource.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the GVK.
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
+
+	// Webhooks holds the Webhooks data
+	Webhooks Webhook `json:"webhooks,omitempty"`
 }
 
-// isEqualTo compares it with another resource
-func (r GVK) isEqualTo(other GVK) bool {
+// Webhook contains information about scaffolded webhooks
+type Webhook struct {
+	Defaulting bool `json:"defaulting,omitempty"`
+	Validation bool `json:"validation,omitempty"`
+	Conversion bool `json:"conversion,omitempty"`
+}
+
+// AddWebhook add the webhook values to the resource with the gkv informed
+func AddWebhook(resource Resource, defaulting, validation, conversion bool) Resource {
+	resource.Webhooks.Defaulting = defaulting
+	resource.Webhooks.Validation = validation
+	resource.Webhooks.Conversion = conversion
+	return resource
+}
+
+// hasSameGKV compares it with another resource
+func (r *Resource) hasSameGKV(other Resource) bool {
 	return r.Group == other.Group &&
 		r.Version == other.Version &&
 		r.Kind == other.Kind
 }
 
 // merge combines fields of two GVKs that have matching group, version, and kind,
 // favoring the receiver's values.
-func (r *GVK) merge(other GVK) {
+func (r *Resource) merge(other Resource) {
 	if r.CRDVersion == "" && other.CRDVersion != "" {
 		r.CRDVersion = other.CRDVersion
 	}
 	if r.WebhookVersion == "" && other.WebhookVersion != "" {
 		r.WebhookVersion = other.WebhookVersion
 	}
+	if !r.Webhooks.isEqualTo(other.Webhooks) {
+		r.Webhooks = other.Webhooks
+	}
```

Comment:
If you run `create webhook --defaulting` then `create webhook --conversion`, the webhook booleans won't be merged correctly. Instead:
```suggestion

	r.Webhooks.merge(other.Webhooks)
```